src/simditor-mention.coffee

```diff
@@ -224,17 +230,33 @@ class SimditorMention extends SimpleModule
 
       $itemEl.appendTo($itemsEl).data 'item',item
 
-    @popoverEl.on 'mouseenter', '.item', (e)->
-      $(@).addClass 'selected'
-        .siblings '.item'
-        .removeClass 'selected'
-    @popoverEl.on 'mousedown touchend','.item', (e)=>
+    @popoverEl.on 'mouseenter', '.item', (e)=>
+      @updateSelectedItem($(e.currentTarget))
+
+    @popoverEl.on 'mousedown','.item', (e)=>
       @selectItem()
       false
 
-    $itemsEl.on 'mousewheel', (e,delta)->
-      $(@).scrollTop $(@).scrollTop() - 10*delta
-      false
+    @popoverEl.on 'touchstart', '.item', (e)=>
+      if !@popoverIsTouching
+        @updateSelectedItem($(e.currentTarget))
+        touches = e.originalEvent.targetTouches[0]
+        @popoverStartY = touches.clientY
+        @popoverIsTouching = true
+        @popoverCancelTouch = false
+      undefined
+    @popoverEl.on 'touchmove', '.item', (e)=>
+      if @popoverIsTouching
+        touches = e.originalEvent.targetTouches[0]
+        currentY = touches.clientY
+        @popoverCancelTouch = @popoverCancelTouch || (Math.abs(@popoverStartY - currentY) > 15)
```

Comment:
It's an arbitrary number based of when I implemented something similar. Goal is to let user move a bit on the screen without cancelling but if they scroll enough we cancel. We can try it on device and tweak it accordingly afterwards!